opennms-services/src/main/java/org/opennms/netmgt/poller/PollerEventProcessor.java

```diff
@@ -726,6 +726,8 @@ private void serviceReschedule(Long nodeId, String nodeLabel, Event sourceEvent,
         for (final Service databaseService : databaseServices) {
             if (polledServices.contains(databaseService)) {
                 LOG.debug("{} is being skipped. Already scheduled.", databaseService);
+                //FIXME Discuss with Jesse
```

Comment:
While investigating another pollerd related bug I came across this same problem. See https://github.com/OpenNMS/opennms/pull/1311.